Automattic/wp-calypso
# Checkout: Convert checkout-thank-you utils to TypeScript

#### Proposed Changes

This converts the post-checkout utility function `getDomainManagementUrl()` to TypeScript. It also removes the function `getActivationCompletedLink()` since it is not used anywhere and might be misleading.

`getActivationCompletedLink()` was [once used by Jetpack Siteless checkout](https://github.com/Automattic/wp-calypso/pull/55897) but that appears to have been removed at some point.

#### Testing Instructions

1. Verify that `getActivationCompletedLink` is not used anywhere.
2. Verify that there are no test failures (which would show a problem with the types).
